internal/ingress/controller/status.go

```diff
@@ -107,7 +108,7 @@ func setupLeaderElection(config *leaderElectionConfig) {
 		LockConfig: resourceLockConfig,
 	}
 
-	ttl := 30 * time.Second
+	ttl := time.Duration(config.ElectionTTL) * time.Second
```

Comment:
Are you sure you need to wrap `config.ElectionTTL` into `time.Duration` here? `time.Duration` is just an alias for `int64` and `time.Second` is one second in nanoseconds represented as `int`, IIRC.